go/vm/evmc/evmc_steppable_interpreter.go

```diff
@@ -62,6 +62,12 @@ func (e *SteppableEvmcInterpreter) StepN(
 		return nil, err
 	}
 
+	evmcBlobHashes := []evmc.Hash{}
+	for _, hash := range params.BlobHashes {
+		evmcBlobHashes = append(evmcBlobHashes, evmc.Hash(hash[:]))
+	}
+	host_ctx.EvmcBlobHashes = evmcBlobHashes
```

Comment:
nit: can you move this up to line 48 to keep the struct initialization together?